docs/USER_GUIDE.ru.md

```diff
@@ -1671,7 +1671,7 @@ spec:
 - Используйте комбинации меток вместо одиночных ограничений. Например, вместо required для одного лейбла (например, env=prod) используйте несколько preferred условий.
 - Учитывайте порядок запуска взаимозависимых ВМ. При использовании Affinity между ВМ (например, бэкенд зависит от базы данных) запускайте сначала ВМ, на которые ссылаются правила, чтобы избежать блокировок.
 - Планируйте резервные узлы для критических нагрузок. Для ВМ с жесткими требованиями (например, AntiAffinity) предусмотрите дополнительные узлы, чтобы избежать простоев при сбое или выводе узла в режим обслуживания.
-- Учитывайте существующие ограничения (`taints`) на узлах.
+- Узлы, на которых запускаются виртуальные машины, не должны иметь каких-либо ограничений по размещению подов (taints).
```

Comment:
This is not a completely correct statement, VMs have tolerations to run on nodes with taints.